src/xeda/flows/flow.py

```diff
@@ -108,6 +108,7 @@ def _validate_quiet(cls, value, values):
         )
         no_console: bool = Field(False, hidden_from_schema=True)
         reports_dir: str = Field("reports", hidden_from_schema=True)
+        checkpoints_dir: str = Field("checkpoints", hidden_from_schema=True)
```

Comment:
Copy+pasting comment chain from the previous PR here:

[kammoh](https://github.com/kammoh) [14 days ago](https://github.com/XedaHQ/xeda/pull/37#discussion_r962021074)
Is this coming from the stale branch or an intended change?

Author
@[makslevental](https://github.com/makslevental) makslevental [7 minutes ago](https://github.com/XedaHQ/xeda/pull/37#discussion_r976679726)
This is intended - [vivado_alt_synth.tcl has set checkpoints_dir `settings.checkpoints_dir`](https://github.com/XedaHQ/xeda/blob/main/src/xeda/flows/vivado/templates/vivado_alt_synth.tcl#L11) but [flow.py does not set this field](https://github.com/XedaHQ/xeda/blob/main/src/xeda/flows/flow.py#L110).

Reply:
Yes, my bad!